server/index.js

```diff
@@ -25,16 +26,18 @@ async function createServer () {
     }
   })
 
-  // Areas and offices change infrequently. Expire cache hourly. Query _should_ take < 2 seconds, allows 3x before error.
-  server.method('db.getAreaToOfficeMap', getAreaToOfficeMap, { cache: { expiresIn: 60 * 60 * 1000, generateTimeout: 6 * 1000 } })
-  // Organisations change infrequently. Expire cache hourly. Query _should_ take < 2 seconds, allows 3x before error.
-  server.method('db.getOrganisationList', getOrganisationList, { cache: { expiresIn: 60 * 60 * 1000, generateTimeout: 6 * 1000 } })
-  // Office locations change infrequently. Expire cache hourly. Query _should_ take < 2 seconds, allows 3x before error.
-  server.method('db.getStandardisedOfficeLocationMap', getStandardisedOfficeLocationMap, { cache: { expiresIn: 60 * 60 * 1000, generateTimeout: 6 * 1000 } })
+  // Areas and offices change infrequently. Expire cache hourly. Query _should_ take < 2 seconds, allows 10 seconds before error.
```

Comment:
I've gone with a consistent 10 sec timeout rather than the 3x of however long it should be taking.